package.json

```diff
@@ -65,7 +65,9 @@
     "format": "prettier --config ./.prettierrc --write './{src,test,scripts}/**/*.{ts,js,json}'",
     "lint": "eslint -c .eslintrc.json --ext .ts ./src ./test",
     "test": "nyc mocha ./test/**/*.test.ts",
-    "package": "yarn pack"
+    "package": "yarn pack",
+    "prettier": "prettier ./**/*.{js,json,ts,md} --write",
+    "format:check": "prettier ./**/*.{js,json,ts,md} --debug-check"
```

Comment:
Ok I added in the prettier and format:check scripts.  

I also ran the prettier script and committed the results, there didn't seem to be too many.